scripts/sandbox/generate.ts

```diff
@@ -244,6 +260,11 @@ if (require.main === module) {
   program
     .description('Generate sandboxes from a set of possible templates')
     .option('--template <template>', 'Create a single template')
+    .option('--includeTemplates <templates>', 'Comma-delimited list of templates to include')
+    .option(
+      '--excludeTemplates <templates>',
+      'Comma-delimited list of templates to exclude. Takes precedence over --includedTemplates'
+    )
```

Comment:
maybe you can change these to be just `--template` and `--skip` or something like that? seems like template and includeTemplates could be merged into a single flag, as includeTemplates could work just as easily for a single template

Reply:
good idea, I changed it to `--templates` and `--exclude` and used the built-in commander functionality instead, so they're now space-delimited lists instead of comma-delimited.